### PR TITLE
add useful_datasets talk

### PR DESCRIPTION
**Pull request by d70-t**

### CODE REVIEW

File: talks/useful_datasets/index.qmd [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+
+{{< video ngc3_selection.mp4 >}}
+
+## hackathons
+
+Output tested on multiple $\mathcal{O}(\textrm{PB})$-scale model runs, 100+ users:
+
+* remarkably little issues raised
+* very positive general feedback
+* enabled diagnostics which seemd impossible before
```
Comment (florianziemen) [inline talks/useful_datasets/index.qmd:335]:
```suggestion
* enabled diagnostics which seemed impossible before
```

**Review by florianziemen — APPROVED** [review]
Looks great!
One typo on the last slide, otherwise all ready for the merge.

File: talks/useful_datasets/index.qmd [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+* index is space-filling curve
+
+# coding the analysis
+
+## dropsonde vs model {auto-animate=true}
+
+Select ICON model output at all<br/>dropsonde locations during [EUREC4A](https://eurec4a.eu/) field campaign:
+
+```{.python}
+sonde_pix = healpy.ang2pix(
```
Comment (lkluft) [inline talks/useful_datasets/index.qmd:255]:
```suggestion
sonde_pix = healpix.ang2pix(
```

**Review by lkluft — APPROVED** [review]
Looks nice! 👍

**Pull request merged**